### PR TITLE
Add description of timescaledb_information.hypertable

### PR DESCRIPTION
**Pull request by svenklemm**
This probably needs some discusion how we want to integrate the view documentation

### CODE REVIEW

File: api.md [hunk 1]
```diff
@@ -25,6 +25,24 @@
 > - [show_tablespaces](#show_tablespaces)
 > - [time_bucket](#time_bucket)
 
+## TimescaleDB informational views [](timescaledb_information)
+
+## timescaledb_information.hypertable [](timescaledb_information.hypertable)
+
+Information about hypertables
```
Comment (dianasaur323) [inline api.md]: Can we match syntax of other helper functions? "Get information about hypertables."

File: api.md [hunk 2]
```diff
@@ -25,6 +25,24 @@
...
+| `schema_name` | Schema name of the hypertable. |
+| `table_name` | Table name of the hypertable. |
+| `table_owner` | Owner of the hypertable. |
+| `num_dimensions` | Number of dimensions. |
+| `num_chunks` | Number of chunks. |
+| `table_bytes` |Disk space used by hypertable |
+| `index_bytes` |Disk space used by indexes|
+| `toast_bytes` |Disk space of toast tables|
+| `total_bytes` |Total disk space used by the specified table, including all indexes and TOAST data|
+
```
Comment (dianasaur323) [inline api.md]: Could we add sample use? 

Get all hypertables. SELECT * ....

Check if a table is a hypertable. SELECT * from ... where table_name = ...

Comment (mfreed) [reply api.md]: Agree with Diana, just to make clear that this is a "view" that one could query through a normal SQL.  Probably obvious to Sven, might not be to everybody.

File: api.md [hunk 3]
```diff
@@ -25,6 +25,24 @@
 > - [show_tablespaces](#show_tablespaces)
 > - [time_bucket](#time_bucket)
 
+## TimescaleDB informational views [](timescaledb_information)
```
Comment (solugebefola) [inline api.md]: Is this meant to be a top level section (i.e. in the sidebar)?

File: api.md [hunk 4]
```diff
@@ -25,6 +25,24 @@
...
+
+## timescaledb_information.hypertable [](timescaledb_information.hypertable)
```
Comment (solugebefola) [inline api.md]: I think a period in the anchor name may mess up the scrolling mechanism.

File: api.md (hunk 2 above)
Comment (solugebefola) [reply api.md]: Yes, there should be a "Sample Usage" section.

File: api.md (hunk 1 above)
Comment (solugebefola) [reply api.md]: Also this should be a header (####) to match the format of other functions.  Maybe "Available columns" as the title?

File: api.md (hunk 3 above)
Comment (svenklemm) [reply api.md]: i wanted feedback on this so i didnt add any sections but diana wanted this to go under utility which is where i put it now

File: api.md [hunk 5]
```diff
@@ -950,6 +951,48 @@ to the server's timezone setting.
...
+| `table_bytes` |Disk space used by hypertable |
+| `index_bytes` |Disk space used by indexes|
+| `toast_bytes` |Disk space of toast tables|
+| `total_bytes` |Total disk space used by the specified table, including all indexes and TOAST data|
+
+#### Sample Usage
+
+Get information about all hypertables.
+
+SELECT * FROM timescaledb_information.hypertable;
```
Comment (solugebefola) [inline api.md]: I think this line can come out.

File: api.md [hunk 6]
```diff
@@ -950,6 +951,48 @@ to the server's timezone setting.
...
+
+```sql
+SELECT * FROM timescaledb_information.hypertable;
+ table_schema | table_name | table_owner | num_dimensions | num_chunks | table_size | index_size | toast_size | total_size 
+--------------+------------+-------------+----------------+------------+------------+------------+------------+------------
+ public       | metrics    | postgres    |              1 |          5 | 99 MB      | 96 MB      |            | 195 MB
```
Comment (solugebefola) [inline api.md]: Just to clarify what a user would get with multiple hypertables in the DB, can you put one or two more example rows here?

**Comment by solugebefola** [comment]
Last thing, I assume this only applies to `1.0`?  If so, can you label the PR?

**Pull request merged**